Makefile

```diff
@@ -143,3 +143,8 @@ crd:
 # Check go format
 check-format: 
 	if [ ! -z "$(FORMAT_LIST)" ] ; then echo invalid format at: ${FORMAT_LIST} && exit 1; fi
+
+generate-helm: wego
```

Comment:
Nice!

Please update the clean target to remove the generated file.

Reply:
Technically, this file is the template to install wego gitops. Why would we want to clean that? It should just exist there and regenerated in case of changes to CRDs or other Manifest files, right?

Reply:
The Chart will end up being stored and then pulled from S3, so there may not be a reason to have it actually live here full-time.

Reply:
👍